service/storage_proxy.cc

```diff
@@ -615,8 +615,11 @@ class storage_proxy::remote {
                     try {
                         // FIXME: get_schema_for_write() doesn't timeout
                         schema_ptr s = co_await get_schema_for_write(schema_version, reply_to_host_id, shard, timeout);
+                        // This erm ensures that tablet migrations wait for replica requests,
+                        // even if the LWT coordinator has been fenced out.
+                        const auto erm = s->table().get_effective_replication_map();
                         // Note: blocks due to execution_stage in replica::database::apply()
```

Comment:
Shouldn't we compare the erm with the fencing token for compatibility?

Perhaps that comes on later.